apps/frontend/src/components/cards/controltable/ControlRowCol.vue

```diff
@@ -40,6 +40,22 @@
       />
       <!-- eslint-enable vue/no-v-html -->
     </v-col>
+    <v-col v-if="'backtrace' in result" cols="12" class="pa-2">
```

Comment:
![image](https://github.com/mitre/heimdall2/assets/2015821/7667135c-d07d-4147-a452-ebf5593e9de0)

backtrace expansion thing shouldn't be there if there is no backtrace provided.  i get that you're trying to do that check in the v-if here but obv it's not working properly.

you should just do object.prototype.hasownproperty: https://stackoverflow.com/a/455340/645647